### ISSUE TITLE
Review observation form

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-watch#204]
1. All labels translated?
2. Dropdowns ok?
3. Errors regarding mandatory fields ok?

**Comment by peterdesmet** [comment]
Blocked by https://github.com/inbo/vespa-watch/issues/134#issuecomment-473347093

**Comment by bartaelterman** [comment]
The errors are not working on fields that are rendered by Vue components: date and address.

**Comment by peterdesmet** [comment]
1. Should be now done in #200

**Comment by timadriaens** [comment]
@peterdesmet @bartaelterman still no luck recording on my phone, can't pintpoint a location on the map. Tried it on my laptop:

- can't pintpoint a location (eg y double clicking) on the map in google chrome, the 'current location' button does work and I can then move the dot to the place I want to record for but this is not ideal nor intuitive
- internet explorer does not display the map under 'location', nor the "date" field
- I also feel it is annoying not to actually see the photo I uploaded (the file name is shown though)

![Vespa-Watch - Microsoft Internet Explorer wordt aangeboden door INBO_2019-04-09_21-22-48](https://user-images.githubusercontent.com/16116743/55828965-d6ebf080-5b0d-11e9-969e-52e32e11606d.jpg)

**Comment by peterdesmet** [comment]
@timadriaens 

- regarding clicking on the map, we won't solve this now
- regarding images: yes, there is now a separate milestone for that https://github.com/inbo/vespa-watch/milestone/4
- regarding internet explorer: I will create a separate issue

Closing this issue: items listed in issue body are ok now
